src/Adapter/Currency/CommandHandler/AddCurrencyHandler.php

```diff
@@ -86,6 +86,7 @@ public function handle(AddCurrencyCommand $command)
             if (!empty($cldrCurrency)) {
                 // The currency may not be declared in the locale, eg with custom iso code
                 $entity->precision = (int) $cldrCurrency->getDecimalDigits();
+                $entity->numeric_iso_code = (int) $cldrCurrency->getNumericIsoCode();
```

Comment:
Shouldn't `getNumericIsoCode()` return `int` instead?

Reply:
In src/Core/Localization/Currency.php, the attribute `$numericIsoCode` is in string, because it stored in 3 characters : https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes#Current_ISO_3166_country_codes

Reply:
What he said 😅